core/src/main/java/tech/tablesaw/api/InstantColumn.java

```diff
@@ -14,6 +14,8 @@
 
 package tech.tablesaw.api;
 
+import static tech.tablesaw.columns.temporal.TemporalPredicates.*;
```

Comment:
Is the formatter doing this automatically?

Reply:
It could be my standard IDE settings. I don't know if Google Format does an override on that. I'm happy to change it here if it's not going to be automatically fixed in the build. 

but why not use *?  

There are places in tablesaw where doing a static import with * makes it easier to use the product. Having access to all the aggregate functions is one example. It kinda sucks if you have to wait for the IDE to figure out which function you want over and over again. 

I'm sure no *s makes sense if you're google and you have 50,000 engineers and you want everything exactly the same, but I think the marginal utility of that rule for the rest of the world is pretty small. I don't think I've ever heard of an actual issue caused by a * import. 

if you feel strongly i'll abide, but we're going to have example code with a half page of imports and that's not a good thing.

Reply:
It is a slight preference, but I definitely do not feel strongly about it. 
The argument against Import * is that it makes library code slightly harder to read when you are on Github (rather than an IDE). I use import * when writing non-library code all the time.